webapp/src/controllers/team.ts

```diff
@@ -82,6 +82,37 @@ export async function inviteTeamMemberApi(req, res) {
 	return dynamicResponse(req, res, 302, { redirect: `/${req.params.resourceSlug}/team` });
 }
 
+/**
+ * @api {delete} /forms/team/invite
+ * @apiName invite
+ * @apiGroup Team
+ *
+ * @apiParam {String} email Email of person to invite
+ */
+export async function deleteTeamMemberApi(req, res) {
+	const { memberId } = req.body;
+	//account with that memberId
+	const memberAccount = await getAccountById(memberId);
+	if(memberAccount) {	
+		const foundTeam = await getTeamById(req.params.resourceSlug);
+		const org = await getOrgById(foundTeam.orgId);
```

Comment:
res.locals.currentOrg

![image](https://github.com/rnadigital/agentcloud/assets/111718755/6356f269-0138-40ad-aa0b-a3dd12264cf6)

Reply:
No can do. It doesn't have the members object, and the reason it doesn't is because the object it comes from is `account` not `org`. I wouldn't change the whole way the auth middleware works just to suite my scenario